torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -234,10 +234,12 @@ def _fill_tensor_shape_type(
     for i, (onnxscript_value, expected_value) in enumerate(
         zip(flat_onnxscript_values, flat_expected_values)
     ):
-        # aten::sym_size output is a int, not a tensor, which stands
-        # for the size of one dim. We treat it as 0-D tensor.
-        # TODO(titaiwang): set shape?
-        if isinstance(expected_value, (torch.SymInt, torch.SymFloat, torch.SymBool)):
+        if expected_value is None:
+            # There is no shape/type from None.
```

Comment:
I would link to the example you shown so readers know when this will happen

Reply:
Would it be possible for use to assume it is always a scalar? Or could there be other cases?

Reply:
hmmm it all depends on what fx.node gives us. This is pure product from fx graph. So I feel like what we should is only taking `None` into consideration of our code.

Reply:
My guess is that for CPU, those outputs are useless or non-generated, so it returns None.